x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/ecs_mapping.json

```diff
@@ -1,12 +1,37 @@
 {
+  "index_patterns": [
+    "try-ecs-*"
+  ],
```

Comment:
This seems weird, but if you're fine with it. I don't think that is going to mess things up, but it's odd to see it in the mapping file like a template would have them.